api/src/test/java/org/openmrs/PersonTest.java

```diff
@@ -375,7 +375,6 @@ public void getAge_shouldGetAgeAfterDeath() {
 		deathDate.set(2000, Calendar.JUNE, 3);
 		Person person = new Person();
 		person.setBirthdate(birthdate.getTime());
-		person.setDead(true);
```

Comment:
My think was that you do not change the existing tests. Hence you leave the call to setDead() as it was. That way we are sure your changes did not break any existing tests.